dist/algoliasearch.helper.js

```diff
@@ -10623,8 +10623,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
    */
-  toggleRefinement: function toggleRefinement(refinementList, attribute, value) {
```

Comment:
What about we still implement `toggleRefinement` as a deprecated method? This way, current implementation won't break. If people were using `toggleFacetRefinement` then it's ok because it will automatically trigger `_toggleFacetRefinement` for conjunctive facets.